src/javascript-node-mongo/devcontainer-template.json

```
@@ -21,7 +21,7 @@
                 "16-buster",
                 "14-buster"
             ],
-            "default": "16-bullseye"
+            "default": "18-bullseye"
```

I know you tried to follow the current style. However, can you change it to 👇  thanks! 

```suggestion
            "default": "18"
```

`18` corresponds to `18-bullseye`. However, the users shouldn't be too much bothered/concerned with `what is -bullseye?` part .